bentoml/pytorch.py

```diff
@@ -318,3 +321,38 @@ def load_runner(
         batch_options=batch_options,
         model_store=model_store,
     )
+
+
+class PytorchTensorContainer(DataContainer[torch.Tensor, torch.Tensor]):
```

Comment:
we still need to add a `DataContainerRegistry.register_container` call here in this file right?

Reply:
my bad